hugo/layouts/partials/header.html

```diff
@@ -54,7 +54,7 @@
                         <a class="btn btn-sm btn-highlight-light" href="https://go.sourced.tech/engine">Request a Demo</a>
                     </li>
                     <li class="nav-item nav-btn">
-                        <a class="btn btn-sm btn-highlight" href="https://docs.sourced.tech/engine/#quickstart">Install Engine</a>
+                        <a class="btn btn-sm btn-highlight" href="https://go.sourced.tech/engine-download">Install Engine</a>
```

Comment:
shouldn't we use the product name?
I was said that we should use `source{d} Engine` always, with no exception.